data/en/entityloadbyexample.json

```diff
@@ -1,16 +1,17 @@
 {
 	"name":"entityLoadByExample",
 	"type":"function",
-	"syntax":"entityLoadByExample(sampleEntity [, unique])",
+	"syntax":"entityLoadByExample(sampleEntity [, unique, matchCriteria])",
 	"returns":"any",
 	"related":["entityLoadByPK"],
-	"description":" Loads and returns an array of objects that match the sampleentity.\nentityLoadByExample(sampleEntity,[unique])",
+	"description":"Loads and returns an array of objects that match the `sampleEntity.",
```

Comment:
You added an opening ` but not a closing one.

Reply:
I fixed it.